streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopicConfig.java

```diff
@@ -38,6 +39,7 @@
 
         this.name = name;
         this.topicConfigs = topicConfigs;
+        this.topicConfigs.put(TopicConfig.MESSAGE_TIMESTAMP_TYPE_CONFIG, "CreateTime");
```

Comment:
Btw: we should (as always) add tests... The test should verify that by default `CreateTime` is set and that a user can overwrite the config.

Reply:
Done, and fixed some broken unit tests. I don't see the Jenkins tests being run on this PR but I ran locally and they're all passing now